raiden/smart_contracts/NettingChannelContract.sol

```diff
@@ -114,37 +111,31 @@ contract NettingChannelContract {
     }
 
     /// @notice Returns the number of blocks until the settlement timeout.
-    /// @return The number of blocks until the settlement timeout.
     function settleTimeout() constant returns (uint) {
         return data.settle_timeout;
     }
 
     /// @notice Returns the address of the token.
-    /// @return The address of the token.
     function tokenAddress() constant returns (address) {
         return data.token;
     }
 
     /// @notice Returns the block number for when the channel was opened.
-    /// @return The block number for when the channel was opened.
     function opened() constant returns (uint) {
         return data.opened;
     }
 
     /// @notice Returns the block number for when the channel was closed.
-    /// @return The block number for when the channel was closed.
```

Comment:
You are also removing comments from the code. This leads me to believe you rebased against an earlier version of master. Please rebase against the latest master.